### DISCUSSION TITLE
PCManFM-Qt - [Option] - Show complete file/dir name

### DISCUSSION THREAD

**Discussion by maplee0** [lxqt/lxqt#2502] [Ideas]
Hello!
As of now long file/dir names are truncated (with ...). How to get full file name below icon? Is it possible? Can't find an option. Thanks!

**Comment by tsujan** [comment]
> As of now long file/dir names are truncated 

It isn't about "now"; it has always been so. It's called eliding, not truncating.

> How to get full file name below icon?

If you want to copy it, you could either use `Tools → Copy Full Path` (or press its shortcut), or open the File Properties dialog.

If you just want to see it, it's in the tooltip. Also, when it isn't too long compared to the window width, it'll be shown on the status-bar if you select the file.

EDIT:
Those who want to always see full names may use the compact view mode.

**Reply by maplee0** [reply]
Thank you for the response!
